.gitignore

```diff
@@ -12,10 +12,6 @@ capybara-*.html
 rerun.txt
 pickle-email-*.html
 
-# TODO Comment out these rules if you are OK with secrets being uploaded to the repo
-config/initializers/secret_token.rb
-config/secrets.yml
-
```

Comment:
Removing this from the Github .gitignore... just for the quick and dirty... get everyone productive approach.